doc/user_guide/postgresql_user_guide.md

```diff
@@ -5,14 +5,41 @@
 ## Uploading the JDBC Driver to EXAOperation
 
 First download the [PostgreSQL JDBC driver](https://jdbc.postgresql.org/).
-Driver version 42.2.6 or later is recommended if you want to establish a TLS-secured connection. 
+Driver version 42.2.6 or later is recommended if you want to establish a TLS-secured connection.
 
 1. [Create a bucket in BucketFS](https://docs.exasol.com/administration/on-premise/bucketfs/create_new_bucket_in_bucketfs_service.htm)
-1. Upload the driver to BucketFS
+2. [Upload the driver to BucketFS](#uploading-a-file-to-bucketfs)
+
+Hint: Put the driver to folder `default/drivers/jdbc/` to make it accessible to [ExaLoader](#configuring-exaloader) as well.
+
+## Uploading a file to BucketFS
+Get password for writing to BucketFS:
+```shell
+CONTAINER=<container id>
+export BUCKETFS_PASSWORD=$(
+  docker exec -it $CONTAINER \
+  grep WritePass /exa/etc/EXAConf \
+  | sed -e 's/.* = //' \
+  | tr -d '\r' \
+  | base64 -d)
+```
+
+Upload the driver file to BucketFS:
+```shell
+FILE=<name of the file to upload>
+FOLDER=<folder in BucketFS>
+curl -v -X PUT -T $FILE http://w:$BUCKETFS_PASSWORD@localhost:2580/$FOLDER/$FILE
+```
+
+| Variable | Description |
+|----------|-------------|
+| `<name of the file to upload>` | Name of the file you want to upload to BucketFS.|
+| `<folder in BucketFS>` | The folder in BucketFS you want the file to upload to. Default root folder is `default`. |
```

Comment:
As much as I like the effort, this does not belong here for two reasons:
1. Customer do not use the docker images in production. They are for testing.
2. This is dialect independent and should therefore be in the central documentation `docs.exasol.com`

Reply:
Generally the documentation already exists, but I recommend you write a DOC ticket to have the doc team add a section about BucketFS in a Docker environment.

Reply:
👍